processing/src/main/java/io/druid/query/search/search/SearchQueryConfig.java

```diff
@@ -27,12 +27,32 @@
  */
 public class SearchQueryConfig
 {
+  public static final String CTX_KEY_STRATEGY = "searchStrategy";
+
   @JsonProperty
   @Min(1)
   private int maxSearchLimit = 1000;
 
+  @JsonProperty
+  private String searchStrategy = AutoStrategy.NAME;
```

Comment:
Okay, I think that's fine, in that case I'll tag this with "release notes" so we can let people know in the notes that there's a new option they can turn on.